src/app/cpr/CMakeLists.txt

```diff
@@ -8,7 +8,7 @@ if(NOT DRISHTI_BUILD_MIN_SIZE)
     target_compile_definitions(train_cpr PUBLIC DRISHTI_USE_IMSHOW)
   endif()
   
-  set_property(TARGET train_cpr PROPERTY FOLDER "app/console")
+  set_target_properties(train_cpr PROPERTIES FOLDER "app/console" INTERPROCEDURAL_OPTIMIZATION DRISHTI_EXE_IPO)
```

Comment:
I will "un-disable" the per target LTO option as a sanity test and can look at this as a possible workaround.  If we can't improve build times significantly, I think LTO may be better left for tagged releases 😕